src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.RandomNumber;
+
+public class Circle extends Shape {
+    private int radius;
+
+    public Circle() {
+        super();
+        this.radius = RandomNumber.getRandomNumber(20);
```

Comment:
how about this comment? 
https://github.com/mate-academy/jv-oop-advanced/pull/14#discussion_r533991229
I meant that we should pass a random value through the constructor:
```
new Circle(RandomNumber.getRandomNumber(20));
```
The same with other figures